### DISCUSSION TITLE
How to change web templates

### DISCUSSION THREAD

**Discussion by elsacutealways** [pocketbase/pocketbase#2330] [Q&A] [answered]
How can I change web templates? 
How can I add custom css to pocketbase pages for resetting password, email confirmation?

**Comment by ganigeorgiev** [comment, marked as answer]
You can create your own email confirmations pages and use the related "Confirm *" apis (_you can find them in the API Preview for your auth collection or in https://pocketbase.io/docs/api-records/#auth-record-actions_).

The email urls can be set via the _Admin UI > Mail settings > {template} > Action URL_ field.
